bootstrap.d/gui-apps.yml

```diff
@@ -0,0 +1,33 @@
+packages:
+  - name: swaybg
+    source:
+      subdir: ports
+      git: 'https://github.com/swaywm/swaybg'
+      branch: 'master'
+      commit: '3cb81c3573036a3606a4c7a1cedbe3e814a22773'
+      version: "1.0"
+    tools_required:
+      - system-gcc
+      - wayland-scanner
+      - host-pkg-config
+      - virtual: pkgconfig-for-target
+        triple: x86_64-managarm
```

Comment:
We shouldn't hardcode any arch here, use `triple: "@OPTION:arch-triple@"` instead.